### ISSUE TITLE
Uploads disabled?

### ISSUE THREAD

**Issue by EMTuckerLab** [globalbioticinteractions/ummzi#2]
I was trying to upload a new/replacement dataset dump, but when I click "upload file" I get the message ""Uploads are disabled. File uploads require push access to this repository."
![image](https://user-images.githubusercontent.com/58006481/108892534-fc892280-75dd-11eb-9567-db51dd575a60.png)
[UMMZI-full-database-export.csv.gz](https://github.com/globalbioticinteractions/ummzi/files/6031203/UMMZI-full-database-export.csv.gz)

**Comment by seltmann** [comment]
@EMTuckerLabUMMZ I updated your permissions and you should get an invite to be part of this repository (via email). Accept invite, log back into Github and try again. Thanks!

**Comment by EMTuckerLab** [comment]
@seltmann Ok - I'll keep watch for the email. Thanks Katja!

**Comment by jhpoelen** [comment]
@EMTuckerLabUMMZ Just in case the email is having trouble getting through, note that you can also check at:

https://github.com/globalbioticinteractions/ummzi/invitations 

for pending invites.

**Comment by EMTuckerLab** [comment]
@jhpoelen Thanks! That was much faster. New dataset is now uploaded. Technically twice- I first uploaded with a slightly different name and then with the same name so it would replace the old...

**Comment by EMTuckerLab** [comment]
Closing issue :)

**Comment by jhpoelen** [comment]
hey @EMTuckerLabUMMZ for some reason, I can't find your recent update in the repository. It says that last update for the data file was three months ago. Can you double check? I am probably not looking in the right location.

**Comment by EMTuckerLab** [comment]
I'm not sure why, but it appears that the project is split in two areas? I
had access to and uploaded the file here:
https://github.com/EMTuckerLabUMMZ/ummzi
But the older files are on a different page (that I can't upload to) here:
https://github.com/globalbioticinteractions/ummzi


~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~
Erika Tucker, PhD
Insect Collection Manager & Assistant Research Scientist
University of Michigan Museum of Zoology
Research Museum Center (RMC), room 1426
3600 Varsity Drive
Ann Arbor, MI 48108
*Office: (734) 647-2199* **email is better at the moment due to mostly
working remotely because of the pandemic**
Email: emtucker@umich.edu
https://prod.lsa.umich.edu/ummz/people/curators-staff/emtucker.html

****PLEASE NOTE:* In response to the ongoing events associated with the
COVID-19 pandemic, the UMMZ Insect Collection's *suspension of
non-essential operations has been extended*.  This includes access to the
collections via scientific visits or outgoing loans.  *Please also DO NOT
ship any specimens (gifts, exchange, loan returns, etc.) to the museum at
this time.*  We apologize for any inconvenience this may cause. Thank you
for your patience and understanding during this challenging time.*****

In the meantime, please check out the following UMMZI digital resources:

*UMMZI Digital Specimen Collection*
https://quod.lib.umich.edu/i/insect2ic
*UMMZI Digital Field Notebook Collection*
https://quod.lib.umich.edu/i/insect1ic
*UMMZI records on GBIF*
https://www.gbif.org/dataset/13e7869e-0c76-473a-a227-53d6e3d6fbf2

**Comment by jhpoelen** [comment]
Yes, that makes sense now. Thanks for clarifying - I was looking in the wrong place.

There's why I was looking in the wrong place:

If you don't have write permissions, it'll create a copy (or fork) in your own organization. So this is what happened before you accepted the invite. And then, you uploaded the new file to your fork, instead of the original repository. 

If you'd like we can switch over to your fork so you can keep you data and GloBI index config at your own lab account. Alternatively, we can stick with the original globalbioticinteractions/ummzi repository. 

Do you have a preference?

**Comment by EMTuckerLab** [comment]
Either way is fine with me - I only have write permissions to the one fork
though. If we work with the other fork I'll need writing permissions... It
worked well earlier when you put the invite in the comment box. I never did
get an email notification (checked junk mail just in case).
Thanks!
Erika

~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~
Erika Tucker, PhD
Insect Collection Manager & Assistant Research Scientist
University of Michigan Museum of Zoology
Research Museum Center (RMC), room 1426
3600 Varsity Drive
Ann Arbor, MI 48108
*Office: (734) 647-2199* **email is better at the moment due to mostly
working remotely because of the pandemic**
Email: emtucker@umich.edu
https://prod.lsa.umich.edu/ummz/people/curators-staff/emtucker.html

****PLEASE NOTE:* In response to the ongoing events associated with the
COVID-19 pandemic, the UMMZ Insect Collection's *suspension of
non-essential operations has been extended*.  This includes access to the
collections via scientific visits or outgoing loans.  *Please also DO NOT
ship any specimens (gifts, exchange, loan returns, etc.) to the museum at
this time.*  We apologize for any inconvenience this may cause. Thank you
for your patience and understanding during this challenging time.*****

In the meantime, please check out the following UMMZI digital resources:

*UMMZI Digital Specimen Collection*
https://quod.lib.umich.edu/i/insect2ic
*UMMZI Digital Field Notebook Collection*
https://quod.lib.umich.edu/i/insect1ic
*UMMZI records on GBIF*
https://www.gbif.org/dataset/13e7869e-0c76-473a-a227-53d6e3d6fbf2

**Comment by jhpoelen** [comment]
@EMTuckerLabUMMZ  Glad to see the UMMZI data coming through. 

(current) Review of [globalbioticinteractions/ummzi] included:
  - 94495 interaction(s)

(previous) Review of [globalbioticinteractions/ummzi] included:
  - 86725 interaction(s)
 
About a 10k additional interactions, you've been busy!

@seltmann do you have a preference for keeping the UMMZI in GloBI's org or at EMTuckerLabUMMZ ?

**Comment by seltmann** [comment]
No preference. But should only be one and if at EMTuckerLabUMMZ @EMTuckerLabUMMZ please invite us as collaborators.

**Comment by jhpoelen** [comment]
@EMTuckerLabUMMZ can you please invite @seltmann and @jhpoelen as collaborators to https://github.com/EMTuckerLabUMMZ/ummzi with "write" permissions? Once that is done, we can deprecate the older repository. 

Please holler if you need help with this.

**Comment by EMTuckerLab** [comment]
@seltmann <https://github.com/seltmann> and @jhpoelen
<https://github.com/jhpoelen>  just added you both as collaborators - I'll
see if I can find the permissions section

~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~
Erika Tucker, PhD
Insect Collection Manager & Assistant Research Scientist
University of Michigan Museum of Zoology
Research Museum Center (RMC), room 1426
3600 Varsity Drive
Ann Arbor, MI 48108
*Office: (734) 647-2199* **email is better at the moment due to mostly
working remotely because of the pandemic**
Email: emtucker@umich.edu
https://prod.lsa.umich.edu/ummz/people/curators-staff/emtucker.html

****PLEASE NOTE:* In response to the ongoing events associated with the
COVID-19 pandemic, the UMMZ Insect Collection's *suspension of
non-essential operations has been extended*.  This includes access to the
collections via scientific visits or outgoing loans.  *Please also DO NOT
ship any specimens (gifts, exchange, loan returns, etc.) to the museum at
this time.*  We apologize for any inconvenience this may cause. Thank you
for your patience and understanding during this challenging time.*****

In the meantime, please check out the following UMMZI digital resources:

*UMMZI Digital Specimen Collection*
https://quod.lib.umich.edu/i/insect2ic
*UMMZI Digital Field Notebook Collection*
https://quod.lib.umich.edu/i/insect1ic
*UMMZI records on GBIF*
https://www.gbif.org/dataset/13e7869e-0c76-473a-a227-53d6e3d6fbf2

**Comment by EMTuckerLab** [comment]
Yeah, I have no idea how to change permissions - or even to see what
permissions you have...

~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~
Erika Tucker, PhD
Insect Collection Manager & Assistant Research Scientist
University of Michigan Museum of Zoology
Research Museum Center (RMC), room 1426
3600 Varsity Drive
Ann Arbor, MI 48108
*Office: (734) 647-2199* **email is better at the moment due to mostly
working remotely because of the pandemic**
Email: emtucker@umich.edu
https://prod.lsa.umich.edu/ummz/people/curators-staff/emtucker.html

****PLEASE NOTE:* In response to the ongoing events associated with the
COVID-19 pandemic, the UMMZ Insect Collection's *suspension of
non-essential operations has been extended*.  This includes access to the
collections via scientific visits or outgoing loans.  *Please also DO NOT
ship any specimens (gifts, exchange, loan returns, etc.) to the museum at
this time.*  We apologize for any inconvenience this may cause. Thank you
for your patience and understanding during this challenging time.*****

In the meantime, please check out the following UMMZI digital resources:

*UMMZI Digital Specimen Collection*
https://quod.lib.umich.edu/i/insect2ic
*UMMZI Digital Field Notebook Collection*
https://quod.lib.umich.edu/i/insect1ic
*UMMZI records on GBIF*
https://www.gbif.org/dataset/13e7869e-0c76-473a-a227-53d6e3d6fbf2

**Comment by jhpoelen** [comment]
Ok - you should be able to see these roles in https://github.com/EMTuckerLabUMMZ/ummzi/settings > manage access . The github configuration ui is useful, but sometimes not as intuitive as you might hope.

**Comment by EMTuckerLab** [comment]
I found the manage access page (where I added you guys as collaborators),
but there doesn't appear to be any settings available to change
writing/reading/etc. ...? I can transfer ownership, but that appears to be
it.

~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~
Erika Tucker, PhD
Insect Collection Manager & Assistant Research Scientist
University of Michigan Museum of Zoology
Research Museum Center (RMC), room 1426
3600 Varsity Drive
Ann Arbor, MI 48108
*Office: (734) 647-2199* **email is better at the moment due to mostly
working remotely because of the pandemic**
Email: emtucker@umich.edu
https://prod.lsa.umich.edu/ummz/people/curators-staff/emtucker.html

****PLEASE NOTE:* In response to the ongoing events associated with the
COVID-19 pandemic, the UMMZ Insect Collection's *suspension of
non-essential operations has been extended*.  This includes access to the
collections via scientific visits or outgoing loans.  *Please also DO NOT
ship any specimens (gifts, exchange, loan returns, etc.) to the museum at
this time.*  We apologize for any inconvenience this may cause. Thank you
for your patience and understanding during this challenging time.*****

In the meantime, please check out the following UMMZI digital resources:

*UMMZI Digital Specimen Collection*
https://quod.lib.umich.edu/i/insect2ic
*UMMZI Digital Field Notebook Collection*
https://quod.lib.umich.edu/i/insect1ic
*UMMZI records on GBIF*
https://www.gbif.org/dataset/13e7869e-0c76-473a-a227-53d6e3d6fbf2

**Comment by jhpoelen** [comment]
I can show you via video chat if you'd like if you have a minute after the TPT webinar.

**Comment by EMTuckerLab** [comment]
sounds good - thanks!

~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~
Erika Tucker, PhD
Insect Collection Manager & Assistant Research Scientist
University of Michigan Museum of Zoology
Research Museum Center (RMC), room 1426
3600 Varsity Drive
Ann Arbor, MI 48108
*Office: (734) 647-2199* **email is better at the moment due to mostly
working remotely because of the pandemic**
Email: emtucker@umich.edu
https://prod.lsa.umich.edu/ummz/people/curators-staff/emtucker.html

****PLEASE NOTE:* In response to the ongoing events associated with the
COVID-19 pandemic, the UMMZ Insect Collection's *suspension of
non-essential operations has been extended*.  This includes access to the
collections via scientific visits or outgoing loans.  *Please also DO NOT
ship any specimens (gifts, exchange, loan returns, etc.) to the museum at
this time.*  We apologize for any inconvenience this may cause. Thank you
for your patience and understanding during this challenging time.*****

In the meantime, please check out the following UMMZI digital resources:

*UMMZI Digital Specimen Collection*
https://quod.lib.umich.edu/i/insect2ic
*UMMZI Digital Field Notebook Collection*
https://quod.lib.umich.edu/i/insect1ic
*UMMZI records on GBIF*
https://www.gbif.org/dataset/13e7869e-0c76-473a-a227-53d6e3d6fbf2

**Comment by jhpoelen** [comment]
@seltmann we figured out that repositories on personal accounts do not have the elaborate role management as the repositories of organization account. We confirmed that we all have edit rights to https://github.com/EMTuckerLabUMMZ/ummzi and decided to do all figure updates there and deprecate (and no longer use) the repository at https://github.com/globalbioticinteractions/ummzi . Also, issues are used to communicate any pending changes or questions.

Thanks y'all for being patient as we figure out all this data management stuff!
